examples/webmcp-remote-assist/browser_test.go

```
@@ -0,0 +1,350 @@
+			chromedp.WSURLReadTimeout(90*time.Second),
+			chromedp.WindowSize(1280, 800),
+		)
+		allocCtx, allocCancel := chromedp.NewExecAllocator(context.Background(), opts...)
+		root, cancel := chromedp.NewContext(allocCtx)
+		if err := chromedp.Run(root); err != nil {
+			browserErr = err
+			cancel()
+			allocCancel()
+			return
+		}
+		browserRoot = root
```

golangci-lint (2.13.2)

[error] nested context in function literal

(fatcontext)